index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ethiopia Poetic Jazz</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/1652c6c81e.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="home" id="home">
+        <nav>
+            <a href="/index.html" class="logo"><span>Ethiopia Poetic Jazz</span></a>
+            <a href="#" class="open-menu"><i class="fas fa-bars"></i></a>
+            <ul class="menu">
+                <li><a href="/index.html" class="home-link">Home</a></li>
+                <li><a href="/about.html" class="about-link">About</a></li>
+                <li><a href="" class="close-menu"><i class="fas fa-times"></i></a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] The top dar nav bar is missing in your design for the desktop version. Kindly note that top navigations divides into a top dark bar and a bottom white bar:
- The top short bar section with social media links, logout, and language links aligned to the right.
- The taller bottom bar with a logo and a list of page links.
- The top bar contains a logo to the left, which you have.
- The top bar contains a list of page links aligned to the right.

Kindly ensure to improve upon this aspect of your design.

**Your top navbars:**
![image](https://user-images.githubusercontent.com/12745474/132686372-221bd45b-cad5-4a3e-a762-9599d5a8ec7e.png)

**The original top navbars:**
![image](https://user-images.githubusercontent.com/12745474/132686563-b9c20ba1-169f-42d3-be46-50803553fae5.png)